requirements.txt

```
@@ -6,7 +6,7 @@ requests==2.22.0
 Click==7.0
 PyYAML~=5.3
 eql~=0.9.5
-elasticsearch~=7.5.1
+elasticsearch~=7.9.1
```

loosen up the minor version
```suggestion
elasticsearch~=7.9
```